### PR TITLE
Feat: k8s 1.24 update

### PR DESCRIPTION
**Pull request by nightfury1204**
### What is the feature/fix?

https://app.asana.com/0/1203637156732418/1204260272257724/f

### Add screenshot or video (optional)
no

### Does it has a breaking change?

yes

### How to use/test it?

- deploy rack of this release 

### Checklist
- [ ] New coverage tests
- [ ] Unit tests passing
- [ ] E2E tests passing
- [ ] E2E downgrade/update test passing
- [ ] Documentation updated
- [ ] No warnings or errors on Deepsource/Codecov

### CODE REVIEW

**Comment by nightfury1204** [comment]
https://github.com/convox/convox/actions/runs/4680686415

File: provider/k8s/k8s.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func FromEnv() (*Provider, error) {
 
 	p := &Provider{
 		Atom:            ac,
-		BuildkitEnabled: os.Getenv("BUILDKIT_ENABLED"),
+		BuildkitEnabled: "true",
```
Comment (yuriadams) [inline provider/k8s/k8s.go:111]: buildkit is it required now?

Comment (nightfury1204) [reply provider/k8s/k8s.go:111]: yes

File: go.mod [hunk 2]
```diff
@@ -1,19 +1,15 @@
 module github.com/convox/convox
 
-go 1.16
+go 1.17
```
Comment (Twsouza) [inline go.mod:3]: awesome job, this will solve some problems in the dependabot PRs :rocket:

File: terraform/fluentd/k8s/containers.conf [hunk 3]
```diff
@@ -77,14 +77,14 @@
...
-	# 	timeout_label @target
-	# </filter>
+    <filter **>
+      @type record_modifier
+      <record>
+        type  k8s
+        log ${record["message"]}
+      </record>
+      remove_keys message
+    </filter>
```
Comment (Twsouza) [inline terraform/fluentd/k8s/containers.conf:87]: why did you uncomment this?

**Review by Twsouza — COMMENTED** [review]
great job, just one question

File: terraform/fluentd/k8s/containers.conf (hunk 3 above)
Comment (nightfury1204) [reply terraform/fluentd/k8s/containers.conf:87]: i didn't uncomment it, i removed that portion and added filter for cri changes
github diff showing it differently

**Review by Twsouza — APPROVED** [review]

**Pull request merged**